assets/app/views/projects.html

```diff
@@ -3,16 +3,46 @@
     <a ng-if="canCreate" href="createProject" style="margin-top: 10px;" class="btn btn-lg btn-primary pull-right">New Project</a>
   </div>
-  <div class="tile tile-project tile-click" style="margin: 10px 0;" ng-repeat="project in projects | orderByDisplayName">
-    <h2 class="project">
-      <a class="tile-target" href="project/{{project.metadata.name}}">{{(project | displayName) || project.metadata.name}}</a>
-      <span  ng-if="project.status.phase != 'Active'" class="pficon-layered" data-toggle="tooltip" data-placement="right" title="This project has been marked for deletion." style="cursor: help; vertical-align: top; margin-left: 5px;">
-        <span class="pficon pficon-warning-triangle"></span>
-        <span class="pficon pficon-warning-exclamation"></span>
-      </span>
-    </h2>
-    <div class="muted" style="margin-top: -5px;" ng-if="project | description">
-      <truncate-long-text content="project | description" limit="512" use-word-boundary="true"></truncate-long-text>
+  <alerts alerts="alerts"></alerts>
+  <div ng-repeat="project in projects | orderByDisplayName">
+    <div class="tile tile-project tile-click" style="margin: 10px 0;">
+      <div class="pull-left" style="max-width:80%">
+        <h2 class="project">
+          <a class="tile-target" href="project/{{project.metadata.name}}">{{(project | displayName) || project.metadata.name}}</a>
+          <span ng-if="project.status.phase != 'Active'" class="pficon-layered" data-toggle="tooltip" data-placement="right" title="This project has been marked for deletion." style="cursor: help; vertical-align: top; margin-left: 5px;">
+            <span class="pficon pficon-warning-triangle"></span>
+            <span class="pficon pficon-warning-exclamation"></span>
+          </span>
+        </h2>
+        <div class="muted" style="margin-top: -5px;" ng-if="project | description">
+          <truncate-long-text content="project | description" limit="512" use-word-boundary="true"></truncate-long-text>
+        </div>
+      </div>    
+      <div ng-if="project.status.phase == 'Active'"
+           class="pull-right action-button" 
+           style="margin:20px; max-width:20%;">
+        <i class="fa fa-trash-o fa-2x" 
+          ng-click="$event.stopPropagation(); toggleConfirm(project)"
+          ng-if="!deletingMap[project.metadata.name] && !confirmingMap[project.metadata.name]" 
+          data-toggle="tooltip" 
+          data-placement="left" 
+          title="Delete Project"></i>
+        <i class="fa fa-strong fa-spinner fa-spin fa-2x" style="color: #000" ng-if="deletingMap[project.metadata.name]"></i>
+      </div>
+      <div class="pull-right" style="padding-top: 20px; margin-right:-20px;" ng-if="confirmingMap[project.metadata.name]">
+        <button class="btn btn-danger" 
+                type="button" 
+                ng-click="$event.stopPropagation(); deleteProject(project);"
+                style="margin-right:20px; color: #fff !important;">
```

Comment:
the delete and cancel buttons look really far apart, maybe try dropping this to 10 px instead of 20?